### PR TITLE
feat: articles screen

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: Feature

This PR resolves [CX-1550]

### Description

This PR introduces the new articles (market news) screen and allows the user to tap into the Market News section on the home screen to open it. 

### Screenshots

https://user-images.githubusercontent.com/4691889/121366932-29b5d280-c93a-11eb-824a-402e8d09eb26.mp4

#### Tablet
| | |
|---|---|
|Simulator Screen Shot - iPad Air (4th generation) - 2021-06-09 at 11 52 33 | Simulator Screen Shot - iPad Air (4th generation) - 2021-06-09 at 11 52 52 | 

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-1550]: https://artsyproduct.atlassian.net/browse/CX-1550

### CODE REVIEW

File: src/lib/utils/track/schema.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export enum PageNames {
   AboutTheFairPage = "AboutTheFair",
   AboutTheShowPage = "AboutTheShow",
   AllArtistSeriesPage = "AllArtistSeries",
+  Articles = "Articles",
```
Comment (pvinis) [inline src/lib/utils/track/schema.ts:89]: Is this used anywhere? If not, let's remove it. We should not add anything new in this file. We should only get rid of things here, and use cohesion things.

**Review by pvinis — CHANGES_REQUESTED** [review]

File: src/lib/Scenes/Articles/ArticlesList.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  refreshing,
+  handleLoadMore,
+  handleRefresh,
+}) => {
+  const numColumns = useNumColumns()
+
+  return (
+    <ProvideScreenTracking
+      info={{
+        context_screen: PageNames.Articles,
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/lib/Scenes/Articles/ArticlesList.tsx:34]: Amazing, that's what we need! I changed to `ProvideScreenTrackingWithCohesionSchema` and got rid of the `schema.ts` change.

File: src/lib/Scenes/Articles/ArticlesList.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+          numColumns={numColumns}
+          key={`${numColumns}`}
+          ListHeaderComponent={() => <ArticlesHeader />}
+          data={articles}
+          refreshControl={<RefreshControl refreshing={refreshing} onRefresh={handleRefresh} />}
+          keyExtractor={(item) => `${item.internalID}-${numColumns}`}
+          renderItem={({ item, index }) => {
+            return (
+              <ArticlesListItem index={index}>
+                <ArticleCardContainer article={item as any} isFluid />
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/lib/Scenes/Articles/ArticlesList.tsx:50]: Good point. `isFlud` (100% width of container) is just used on this screen. It's used two times without `isFluid` in article rails (home screen, artist screen).

[the remark this answers is not in the document]
Comment (pvinis) [reply src/lib/Scenes/Articles/ArticlesList.tsx]: cool then. I was thinking if it's needed or not, and it seems it is needed. thanks :)

File: src/lib/utils/track/schema.ts (hunk 1 above)
Comment (olerichter00) [reply src/lib/utils/track/schema.ts]: I removed it.

**Review by leamotta — APPROVED** [review]
Looks good!

**Comment by olerichter00** [comment]
#squashongreen

**Pull request merged**